tests/RegressionTests.hs

```diff
@@ -309,6 +312,10 @@ nativeOpts :: Options
 nativeOpts = defaultOptions{useNativeArrays=True}
 nativeRetOpts :: Options
 nativeRetOpts = defaultOptions{useNativeReturns=True}
+woolOptions :: Options
```

Comment:
The naming convention for variables for Options is "Opts" in this part of the code.

Reply:
Ok.